shared/macros-oval.jinja

```diff
@@ -218,6 +218,32 @@
   </unix:file_test>
 {{%- endmacro %}}
 
+{{#
+  Macro to define the OVAL criterion that requires a file not to exist.
+  This macro can take one parameter:
+    - filepath (String): Path to the file to be checked.
+    - stem of the test name
+#}}
+{{%- macro oval_config_file_absent_criterion(filepath, stem) -%}}
+	<criterion comment="Pass if {{{ filepath }}} does not exist" test_ref="test_{{{ stem }}}" />
```

Comment:
This macro is inappropriate. Please remove it and remove all of its occurrences.

Reply:
Why? We have already a similar concept used successfully elsewhere, see e.g. the `oval_config_file_exists_criterion` macro and more like that in the OVAL jinja file.

Reply:
Yes, please remove that as well.